developer/style-conventions.rst

```diff
@@ -0,0 +1,120 @@
+Style Conventions
+=================
+
+We are thankful that you took your time to contribute to this community.
```

Comment:
We already state this concept in other pages, therefore this line is not necessary, please remove it